pulp_2to3_migration/tests/functional/test_file_migration.py

```diff
@@ -133,7 +133,7 @@ def setUpClass(cls):
 
     def _do_test(self, repos, migration_plan):
         mp = self.migration_plans_api.create({'plan': migration_plan})
-        mp_run_response = self.migration_plans_api.run(mp.pulp_href, data={})
+        mp_run_response = self.migration_plans_api.run(mp.pulp_href, {})
```

Comment:
@dkliban and @dralley

Reply:
the `data` is a workaround from drf-yasg:
https://github.com/axnsan12/drf-yasg/blob/master/src/drf_yasg/inspectors/view.py#L144-L150